integrations/vercel.mdx

```diff
@@ -1,4 +1,110 @@
 ---
-title: "Vercel"
-url: "https://github.com/onkernel/vercel-template"
----
\ No newline at end of file
+title: "Vercel Marketplace"
+description: "Integrate Kernel through the Vercel Marketplace"
+---
+
+## Integrate Kernel through the Vercel Marketplace
+
+The Vercel Marketplace allows Vercel users to install and configure third-party services, like Kernel, directly from the Vercel dashboard. Kernel offers an [official integration](https://vercel.com/marketplace/kernel) that Vercel users can install to integrate Kernel into their Vercel projects.
+
+### Installing the Kernel Integration
+
+{/* <Tip>
+  Deploy an example Next.js application and install the Kernel integration with our [template](https://github.com/onkernel/vercel-template).
+</Tip> */}
+
+1. Navigate to the [Kernel integration](https://vercel.com/marketplace/kernel) in the Vercel Marketplace.
+2. Click **Install** to add Kernel to your Vercel team.
+3. Select your pricing plan.
+4. Provide a name for your Kernel installation. (this is not used for billing or anything else)
+5. Click **Create**.
+
+Vercel will automatically provision a Kernel account, organization, and API key for you. Vercel calls the provisioned account, organization, and API key a **Resource**.
```

Comment:
You are talking about User accounts here and in the next paragraph, right? It may help to say "Kernel User account" instead of just "Kernel account". I get a little mixed up thinking of Kernel Organizations vs. Kernel accounts, so adding "User" may help (me at least).